authentication-models/src/test/java/com/alexkudlick/authentication/models/AuthenticationRequestTest.java

```diff
@@ -25,15 +26,17 @@ public void testDeserialize() throws IOException {
     @Test
     public void testSerialize() throws IOException {
         AuthenticationRequest request = new AuthenticationRequest("testUserName", "testPassword");
-        String expectedJson = "{" +
-            "\"userName\":\"testUserName\"," +
-            "\"password\":\"testPassword\"" +
-        "}";
+        String expectedJson1 = "{" +
+                "\"userName\":\"testUserName\"," +
+                "\"password\":\"testPassword\"" +
+                "}";
+        String expectedJson2 = "{" +
+                "\"password\":\"testPassword\"," +
+                "\"userName\":\"testUserName\"" +
+                "}";
         String serialized = new ObjectMapper().writeValueAsString(request);
-        assertEquals(
-            expectedJson,
-            serialized
-        );
+        Boolean assertionVal=expectedJson2.equals(serialized) || expectedJson1.equals(serialized);
```

Comment:
This can simply be `boolean`, correct?

Reply:
Yes, that can just be `boolean` instead of `Boolean`, I will change that.
Do you think I could use the boolean statement directly inside the assertion instead of having a variable for it like -
`assertTrue(expectedJson2.equals(serialized) || expectedJson1.equals(serialized);` 
instead of `assertTrue(assertionVal);`

Reply:
Yea, that would be simpler

Reply:
I have made the changes